packages/icons/src/icon-system/collection.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 16 16" fill-rule="evenodd"><path fill="none" d="M0 0h16v16H0z"/><g fill="#1b2029"><rect x="1" y="1" width="6" height="6" rx="1"/><rect x="1" y="9" width="6" height="6" rx="1"/><rect x="9" y="1" width="6" height="6" rx="1"/><rect x="9" y="9" width="6" height="6" rx="1"/></g></svg>
```

Comment:
looks like there are some hardcoded `fill` values in here. Is that what we want? should it inherit from the CSS?

Reply:
Yes, thank you for noticing that. There have been some issues with the icons provided for this, I had to manually add the viewBox.